index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Roland Portfolio</title>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+
+    <body>
+        <div class="wrapper" id="wrapper">
+            <header class="header">
+                <nav class="nav">
+                    <img src="img/image_geometry_1.svg" alt="">
```

Comment:
- [ ] Well done on adding the image but this is a background image so kindly change it to the background and not the img tag. Keep coding 💪